### DISCUSSION TITLE
please_donate/

### DISCUSSION THREAD

**Discussion by giscus** [Micraow/chat#2] [Announcements]
# please_donate/

你好，本站的访客： pengs.top历经了无数波折（从msblog.ml迁移,注册域名，备案（还要... - Micraow - Peng's Blog

https://pengs.top/please_donate/

**Comment by Geeker-LStar** [comment]
我是李天星~
给你转了 50（微信），你应该收到了叭~
不知道你现在攒到多少钱了，如果不够的话联系我，200 多这个价格我可以帮你解决哒~
加油加油！祝你的网站越做越好！嘿嘿~！

**Reply by Micraow** [reply]
谢谢！的确已经收到了，让我比较意外的是，现在居然已经筹集到了143.6元钱了，剩下来的一部分，我的零花钱还是够的。我准备过两天期末考完试之后找个同学帮我把这笔资金转移到支付宝，因为商家不支持微信😂

另外，也祝你那边一切顺利！
